src/theme/widgets/map/map.scss

```diff
@@ -0,0 +1,53 @@
+.map {
+  min-height: 630px;
+  position: relative;
+  .map__window {
+    background: $card-title-background-color;
+    .gm-style {
+      & > div:nth-of-type(2), & > .gmnoprint:not(.gm-bundled-control), & > .gm-style-cc {
+        display: none;
+      }
+    }
+  }
+  .mdl-card__supporting-text {
+    min-height: 185px;
+  }
+  h5 {
+    margin-top: 0;
+  }
+  div > a {
+    font-size: 14px;
+  }
+  .map__target-location, .map__target-opening-hours {
+    margin-bottom: 10px;
+    & > * {
+      vertical-align: middle;
+    }
+    .material-icons {
+      color: unquote("rgb(#{$color-primary})");
+      margin-top: -2px;
+      margin-right: 5px;
+      font-size: 20px;
+    }
+  }
+  .map__target-image {
+    background: $card-title-background-color;
+    border-radius: 50%;
+    width: 170px;
+    height: 170px;
+    margin: 0 10px 5px 15px;
+  }
+  .map__target-picture {
+    height: 100%;
+    transition: opacity 0.3s;
+    background: url("/assets/images/map_target_images_sprite.png") no-repeat top left;
```

Comment:
you have `$image_path`. Why don't you use it?

Reply:
✔️